src/estimation/src/BerdyHelper.cpp

```diff
@@ -279,6 +284,14 @@ bool BerdyHelper::initSensorsMeasurements()
         m_nrOfSensorsMeasurements += 6;
     }
 
+    // Add CoM accelerometer sensor
+    if (m_options.includeCoMAccelerometerAsSensor)
+    {
+        //std::cout << "Berdy helper : Considering CoM accelerometer sensor in initSensorsMeasurements" << std::endl;
```

Comment:
If this print is not significant, you may consider to remove it.

Reply:
Done in https://github.com/robotology/idyntree/pull/571/commits/f57ab3c061b91f83c5351d671132cc922ca75556